transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java

```diff
@@ -377,11 +378,15 @@ protected void run() {
                         break;
 
                     case SelectStrategy.SELECT:
-                        if (wakenUp.get() == 1) {
-                            wakenUp.set(0);
-                        }
-                        if (!hasTasks()) {
-                            strategy = epollWait();
+                        wakenUp.lazySet(0);
```

Comment:
can you add a comment why `lazySet` ?

Reply:
@normanmaurer have now added this comment and rebased